moban/constants.py

```diff
@@ -12,7 +12,7 @@
     ".%s%s" % (PROGRAM_NAME, DEFAULT_YAML_SUFFIX),
     ".%s%s" % (PROGRAM_NAME, ".yaml"),
 ]
-DEFAULT_TEMPLATE_TYPE = "jinja2"
+DEFAULT_TEMPLATE_TYPE = "jj2"
```

Comment:
I believe this should be reverted back to `"jinja2"`, so the template types are engine names, not file extensions.

coala has four commonly used extensions [registered](https://github.com/coala/coala/blob/master/coalib/bearlib/languages/definitions/Jinja2.py) for jinja2

Many of the more common engines have lots of file extensions, and derived slightly different implementations, like .hbs for handlebars is actually almost Mustache, so users would likely want to use our Mustache engine for .hbs files.